src/Sylius/Bundle/ApiBundle/Resources/config/services/controllers.xml

```diff
@@ -23,6 +23,11 @@
             <argument type="service" id="sylius.repository.order_item" />
         </service>
 
+        <service id="Sylius\Bundle\ApiBundle\Controller\GetCustomerStatisticsAction">
+            <argument type="service" id="sylius.command_bus" />
+            <tag name="controller.service_arguments" />
```

Comment:
Should controller use serializer directly instead? Other api controllers don't use tag.

Like
![image](https://github.com/Sylius/Sylius/assets/870747/83bf3b7f-e4bb-49e6-abe7-db008633fc31)
![image](https://github.com/Sylius/Sylius/assets/870747/08083755-1ce7-46cc-9055-8d9ce16feafe)

Please sync implementation with `src/Sylius/Bundle/ApiBundle/Controller/GetStatisticsAction.php` in parallel PR https://github.com/Sylius/Sylius/pull/15598/files

Reply:
```suggestion
```

So is controller tag needed? Other controllers don't use it.